cockatrice/src/window_main.cpp

```diff
@@ -257,6 +257,8 @@ void MainWindow::actSettings()
 
 void MainWindow::actExit()
 {
+    if (QMessageBox::question(this, tr("Close Cockatrice"), tr("Are you sure you want to close Cockatrice?"), QMessageBox::Yes | QMessageBox::No, QMessageBox::No) != QMessageBox::Yes)
```

Comment:
Yes, because if you close on Mac from the menu it needs this dialog 

> On Jul 15, 2015, at 5:20 PM, ctrlaltca <notifications@github.com> wrote:
> 
> In cockatrice/src/window_main.cpp:
> 
> > @@ -257,6 +257,8 @@ void MainWindow::actSettings()
> >  
> >  void MainWindow::actExit()
> >  {
> > +    if (QMessageBox::question(this, tr("Close Cockatrice"), tr("Are you sure you want to close Cockatrice?"), QMessageBox::Yes | QMessageBox::No, QMessageBox::No) != QMessageBox::Yes)
> is this supposed to still be here?
> 
> —
> Reply to this email directly or view it on GitHub.
>